Nxo feature (#570)
braintree/braintree_android: CHANGELOG.md

# Braintree Android SDK Release Notes

## unreleased
* Adds `invalidateClientToken` method to `BraintreeClient`, which invalidates the cached client
token authorization provided by the `ClientTokenProvider`
* Add `isExplicitCancelation` parameter to `UserCanceledException`
* Trim tokenization key and client token before parsing
* PayPalNativeCheckout

>   * Adding in [PayPalNativeCheckout] module to use the native checkout for PayPal

Reviewer: ```suggestion
  * Add new module to use Native Checkout for PayPal transactions
```

Reviewer: Nitpick. We rarely add new modules so we don't have a set practice for announcing new features via CHANGELOG.
